### ISSUE TITLE
Configuration file not loaded unless explicitly provided via CLI

### ISSUE THREAD

**Issue by JoshuaHall** [alacritty/alacritty#3344]
### System

OS: Windows
Version: alacritty 0.4.1 (4d9253b)
Windows: ConPTY

### Logs (`alacritty -v`)

```
[2020-02-15 12:21] [INFO] Welcome to Alacritty
[2020-02-15 12:21] [INFO] Device pixel ratio: 1.25
[2020-02-15 12:21] [INFO] Initializing glyph cache...
[2020-02-15 12:21] [INFO] ... finished initializing glyph cache in 0.0420503s
[2020-02-15 12:21] [INFO] Cell Size: 10 x 21
[2020-02-15 12:21] [INFO] Padding: 0 x 0
[2020-02-15 12:21] [INFO] Width: 1422, Height: 700
[2020-02-15 12:21] [INFO] PTY Dimensions: Line(33) x Column(142)
[2020-02-15 12:21] [INFO] Using ConPTY backend
[2020-02-15 12:21] [INFO] Initialisation complete
```

### Description

Potentially relevant note: I installed alacritty using `scoop` on Windows.

When I went to change my default shell from Powershell to git bash/wsl/bash, my configuration file wasn't located at `%APPDATA%\alacritty\alacritty.yml`, and there was no `%APPDATA%\alacritty` directory created. I created the directory and then copied the `alacritty.yml` directly from Github. After starting Alacritty again, it still didn't recognize my configuration file. I then manually provided the configuration file path with `alacritty --config-file <my_path>`, which DID work. However, I still can't figure out how to get alacrity to load the configuration file from the default configuration file location without having to manually launch it with a command line argument. Any help would be appreciated.

**Comment by kchibisov** [comment]
>When I went to change my default shell from Powershell to git bash/wsl/bash, my configuration file wasn't located at %APPDATA%\alacritty\alacritty.yml, and there was no %APPDATA%\alacritty directory created.

if you read this [section](https://github.com/alacritty/alacritty#configuration) , you can notice "Alacritty doesn't create the config file for you" line.

`%APPDATA% is your environment variable, so make sure you place your config file in a right place. Like you shouldn't putt it in directory called `%APPDATA%` or `AppData`, since env variable could be pointing to a bit different location.

 Most of the time it's `\AppData\Roaming\`.

**Comment by JoshuaHall** [comment]
After changing the configuration file location to that it finally works. Thanks! I should have known since going to `%APPDATA%` took me to `\AppData\Roaming\`, but I put it in `\AppData\`. Seems like a rather confusing system 😅

**Comment by JadedEvan** [comment]
Thanks for the specific details @JoshuaHall - this was helpful when I stumbled on this issue. As a fellow Windows 10 user - I had to place my configuration in `\AppData\Roaming\alacritty\alacritty.yml`. I tried putting this in `\AppData` as mentioned above but it did not work until it went into `Roaming` subdirectory. Also worth noting the file extension must be `.yml` and not `.yaml` - many of the color scheme repos around right now are using the four letter extension which was another point of confusion.

I had the similar experience where the `alacritty/alacritty.yml` were not created for me by default on install. This is running alacritty `0.4.3 (c723247)` through Windows 10 Powershell.
